### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites-Create Web Site_Labels or Instructions: No label provided for the directory path input field.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#139]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**MAS ID Violated:** MAS3.3.2

**Repro Steps:** 
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. Tab till "Create Web Site" expandable button and Press Enter.
6. Tab till "Select Folder" button and Press Enter.
7. Verify that appropriate label is provided for all the input fields.

**Actual Result:**
No label provided for the directory path input field.

**Expected Result:**
An appropriate label should be provided for the directory path input field.

**Note**: This issue is observed for all the similar input fields.

**User Impact:**
When label elements are associated with input elements the label is spoken by screen readers when the field receives focus and users with impaired motor control are helped by a larger clickable area for the control, since clicking on the label or the control will activate the control.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={4b95a70d-258c-474a-820e-5c8b79d5c0bf}

**Attachment:**
![bug 139_no label provided for the directory path input field](https://user-images.githubusercontent.com/38521133/44723999-2df90400-aaef-11e8-8657-fbcc27e05ca4.png)

**Comment by jhkimnew** [comment]
Fixed with adding Aria-label

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
